test/CometWrapper.t.sol

```diff
@@ -255,78 +482,90 @@ contract CometWrapperTest is BaseTest, CometMath {
         uint256 totalPrincipal = unsigned256(comet.userBasic(address(cometWrapper)).principal);
         assertEq(cometWrapper.totalSupply(), totalPrincipal);
     }
 
-    function test__transferFromWorksForSender() public {
+    function test_transferFromWorksForSender() public {
         vm.startPrank(alice);
         comet.allow(wrapperAddress, true);
         cometWrapper.mint(5_000e6, alice);
 
         cometWrapper.transferFrom(alice, bob, 2_500e6);
-        assertApproxEqAbs(cometWrapper.balanceOf(alice), 2_500e6, 1);
         vm.stopPrank();
+
+        // TODO: investigate why this gets rounded down...i think because mint is rounded down. need to verify
+        assertApproxEqAbs(cometWrapper.balanceOf(alice), 2_500e6, 1);
+        assertEq(cometWrapper.balanceOf(bob), 2_500e6);
     }
 
-    function test__transferFromRespectsAllowances() public {
+    function test_transferFromUsesAllowances() public {
         vm.startPrank(alice);
         comet.allow(wrapperAddress, true);
         cometWrapper.mint(5_000e6, alice);
         vm.stopPrank();
 
         // Need approvals to transferFrom alice to bob
-        vm.startPrank(bob);
-        vm.expectRevert(CometHelpers.LackAllowance.selector);
+        vm.prank(bob);
+        vm.expectRevert(CometHelpers.InsufficientAllowance.selector);
         cometWrapper.transferFrom(alice, bob, 5_000e6);
-        vm.stopPrank();
 
         vm.prank(alice);
-        cometWrapper.approve(bob, 2_500e6);
+        cometWrapper.approve(bob, 2_700e6);
 
         vm.startPrank(bob);
         // Allowances should be updated when transferFrom is done
-        assertEq(cometWrapper.allowance(alice, bob), 2_500e6);
+        assertEq(cometWrapper.allowance(alice, bob), 2_700e6);
         cometWrapper.transferFrom(alice, bob, 2_500e6);
         assertApproxEqAbs(cometWrapper.balanceOf(alice), 2_500e6, 1);
-        assertApproxEqAbs(cometWrapper.balanceOf(bob), 2_500e6, 1);
+        assertEq(cometWrapper.balanceOf(bob), 2_500e6);
 
-        vm.expectRevert(CometHelpers.LackAllowance.selector);
+        // Reverts if trying to transferFrom again now that allowance is used up
+        vm.expectRevert(CometHelpers.InsufficientAllowance.selector);
         cometWrapper.transferFrom(alice, bob, 2_500e6);
         vm.stopPrank();
-        assertEq(cometWrapper.allowance(alice, bob), 0);
+        assertEq(cometWrapper.allowance(alice, bob), 200e6);
 
         // Infinite allowance does not decrease allowance
         vm.prank(bob);
         cometWrapper.approve(alice, type(uint256).max);
+        assertEq(cometWrapper.allowance(bob, alice), type(uint256).max);
 
         vm.startPrank(alice);
         cometWrapper.transferFrom(bob, alice, 1_000e6);
         assertEq(cometWrapper.allowance(bob, alice), type(uint256).max);
         vm.stopPrank();
     }
 
-    function test__transferFromRevertsOnLackingAllowance() public {
+    function test_transferFrom_revertInsufficientAllowance() public {
```

Comment:
nit: `test_transferFromRevertInsufficientAllowance` to be consistent with your other names

Reply:
Updated the other test names to be consistent with this one.